cypress/e2e/ui-tests/test-public-apps-page.cy.js

```diff
@@ -8,6 +8,7 @@ describe("Test of the public apps page", () => {
     it("should contain header with text Apps", () => {
 
         cy.get('h3').should('contain', 'Apps')
+        cy.get("title").should("have.text", "Overview | SciLifeLab Serve")
```

Comment:
Should the apps view really have title "Overview | SciLifeLab Serve"?

Reply:
I think that's the question to @akochari 
I think `Apps` would have been fine

Reply:
@akochari What do you think?

Reply:
Yes, agree, should be Apps.